static/app/views/settings/project/server-side-sampling/serverSideSampling.tsx

```diff
@@ -395,6 +401,10 @@ export function ServerSideSampling({project}: Props) {
 
   const uniformRule = rules.find(isUniformRule);
 
+  if (loading) {
+    return <LoadingIndicator />;
+  }
```

Comment:
Do we now want to block the whole page for those 2 slow requests?

Reply:
Currently, most settings views use the `AsyncComponent`, which blocks the whole page until the requests are completed, so I thought we would like to keep it consistent.

I will update here and maybe in the future, we can update everywhere else too... its better not to block :)

Reply:
I agree, better not to block and render what we can (rules table).

If we merge like this then the whole `ServerSideSamplingStore` work was for nothing and the page will be spinning for a long time.